commands/daemon_notwin.go

```diff
@@ -1,39 +1,42 @@
-package main
```

Comment:
lets call it daemon_linux.go instead

Reply:
Agreed on both parts. I'll make revisions today.